language/sl.json

```diff
@@ -503,8 +503,8 @@
           "default": "Skrij informacije."
         },
         {
-          "label": "Code contents (Text for Readspeakers)",
-          "default": ". Code contents"
+          "label": "Vsebina kode (besedilo za bralnik zaslona)",
+          "default": "Vsebina kode."
```

Comment:
@miropuhek The `.` was not misplaced in the original text. It ensures that the screen reader will pause briefly before readiny "Code contents".